mock_hardware/include/mock_components/generic_system.hpp

```diff
@@ -100,6 +99,37 @@ class HARDWARE_INTERFACE_PUBLIC GenericSystem : public hardware_interface::Syste
   std::vector<std::vector<double>> gpio_commands_;
   std::vector<std::vector<double>> gpio_states_;
 
+  double actuator_slowdown_;
```

Comment:
do we need this? If yes, for what exactly? Between joint and actuator there if always some factor to calculate values i transmission.

Reply:
removed it for now its not directly related to transmissions